examples/js-libp2p-example-delegated-routing/package.json

```diff
@@ -20,17 +20,17 @@
     "test": "npm run build && test-browser-example test"
   },
   "dependencies": {
-    "@helia/delegated-routing-v1-http-api-client": "^3.0.1",
-    "@helia/delegated-routing-v1-http-api-server": "^3.0.3",
-    "libp2p": "^1.0.8",
+    "@helia/delegated-routing-v1-http-api-client": "^4.0.0",
+    "@helia/delegated-routing-v1-http-api-server": "^4.0.0",
+    "libp2p": "^2.0.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-scripts": "^5.0.1"
   },
   "devDependencies": {
     "test-ipfs-example": "^1.0.0",
     "eslint-config-ipfs": "^7.0.2",
-    "helia": "^4.2.4"
+    "helia": "next"
```

Comment:
:eyes:

Reply:
This is necessary because the changes here need `helia@5.x.x` - this can be updated after https://github.com/ipfs/helia/issues/586